cmd/testwrapper/testwrapper.go

```diff
@@ -119,19 +122,28 @@ func runTests(ctx context.Context, attempt int, pt *packageTests, otherArgs []st
 			// ignore
 		case "run":
 			resultMap[name] = &testAttempt{
-				name: name,
+				Name: name,
 			}
 		case "skip", "pass", "fail":
-			resultMap[name].outcome = goOutput.Action
+			resultMap[name].Outcome = goOutput.Action
 			out = append(out, resultMap[name])
 		case "output":
 			if strings.TrimSpace(goOutput.Output) == flakytest.FlakyTestLogMessage {
-				resultMap[name].isMarkedFlaky = true
+				resultMap[name].IsMarkedFlaky = true
 			} else {
 				resultMap[name].logs.WriteString(goOutput.Output)
 			}
 		}
 	}
+	f, err := os.Create("test_attempts.json")
```

Comment:
@clairew you want to make this use the `attempt` var too otherwise reruns will clobber the output.